modules/programs/nheko.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nheko;
+
+  iniType = with types;
+    let
+      primitiveType = either str (either bool (either int float));
+      multipleType = either primitiveType (listOf primitiveType);
+      sectionType = attrsOf multipleType;
+      supersectionType = attrsOf (either multipleType sectionType);
+    in attrsOf supersectionType;
```

Comment:
This looks like a pretty standard INI file... we should be using `pkgs.formats.ini`

(See `modules/services/pantalaimon.nix` as an example)

Reply:
Oh nice, wasn't aware of that, I'll check it out!